client/src/pages/Home.js

```diff
@@ -43,24 +85,43 @@ const StyledP = styled.p`
   line-height: 2.1rem;
   letter-spacing: 0.1px;
   margin: 0;
+  
+  @media only screen and (min-width: 600px){
+    //theme.typography.heading.one is not enough size
+    font-size: 1vw;
+    text-align: left;
+    color: white;
+  }
 `;
 
 const OnboardingContainer = styled.div`
   margin-top: 4rem;
+
+  @media only screen and (min-width: 600px){
+    display: "flex";
+    flexWrap: "wrap";
+    width: 100%;
+    margin: auto 0;
+  }
 `;
 
 export const Home = (props) => {
   console.log("render home", { props });
   return (
-    <div className="text-center">
+    <MainContainer className="text-center">
```

Comment:
I also noticed this class while trying to clean up the styles, wonder if we still need these classes defined globally or if there is a better way if we use StyledComponents with something reusable as a component perhaps?

Reply:
I replaced with styled-component